bftengine/src/bftengine/PersistentStorageDescriptors.cpp

```diff
@@ -261,7 +274,8 @@ void DescriptorOfLastNewView::deserializeElement(uint32_t id, char *buf, size_t
   actualSize = 0;
   ConcordAssert(id < viewChangeMsgsNum);
 
-  auto *msg = MessageBase::deserializeMsg(buf, bufLen, actualSize);
+  std::unique_ptr<MessageBase> msgBase(MessageBase::deserializeMsg(buf, bufLen, actualSize));
+  auto msg = new ViewChangeMsg(msgBase.get());
   ConcordAssert(viewChangeMsgs[id] == nullptr);
   viewChangeMsgs[id] = ((ViewChangeMsg *)msg);
```

Comment:
Nit: We can remove the cast now as `msg` is of the proper type.

Reply:
Thanks @dartdart26 ! Will remove it.